src/cls/queue/cls_queue_src.cc

```diff
@@ -327,11 +327,13 @@ int queue_list_entries(cls_method_context_t hctx, const cls_queue_list_op& op, c
     }
 
     //If there is leftover data from previous iteration, append new data to leftover data
-    uint64_t entry_start_offset = start_offset - bl.length();
+    //NOLINTBEGIN(bugprone-use-after-move)
+    uint64_t entry_start_offset = start_offset - bl.length();  
     CLS_LOG(20, "INFO: queue_list_entries(): Entry start offset accounting for leftover data is %lu", entry_start_offset);
     bl.claim_append(bl_chunk);
     bl_chunk = std::move(bl);
-
+    bl.clear();
```

Comment:
Since we're clearing it here, and bl isn't used outside the loop that I can see, would it make more sense just to move its declaration to the top of the loop rather than add the clear and annotations?

Reply:
@adamemerson done